xlsum card preparer, with signature and with debug = False

dafnapension opened the pull request:
fixed the signature issue, by recommitting.
Applied yoav's comments re. debug = False.

dafnapension commented:
Hi @yoavkatz ,  I applied here all your comments from  #290, for same branch that I committed then without signature.
Here I signed all, and set debug = False, and added the generated json cards.
Please approve and merge.

yoavkatz reviewed (changes requested):
Can you change "text"->"document" and "target"->"summary" 

And also add a "text_type" field with value "text".

This will allow it to reuse other summarization templates (once we add them).

dafnapension commented:
Hi @yoavkatz , I made the changes you suggested.
Please review and approve, if you see fit, and merge.
Thanks!

yoavkatz commented on prepare/cards/xlsum.py:
```diff
@@ -0,0 +1,40 @@
+            ]
+        ),
+    )
+    if lang == "english":
```
```suggestion
    if lang == langs[0]
```

yoavkatz replied:
The reason is , is that we don't want to add to catalogs cards, before at least one is tested.  And we don't know if English is the first.

dafnapension replied:
Thanks, @yoavkatz , this is a very nice thought, that did not occur to me.
Fixing right away!

dafnapension replied:
Thanks again, @yoavkatz , fixed this, and also for Amazon from yesterday.
Please see if you can approve now.

dafnapension commented:
Hi @yoavkatz , I fixed per your suggestion, and updated merge into this branch, to avoid the conflicts that grew up..
Please review, and if you see fit - approve and merge.

yoavkatz commented:
Approved. You can merge,

dafnapension commented:
Thank you, @yoavkatz , for all the lessons I learned from you
